tools/migration/ppaas-artifact-converter/src/main/java/org/apache/stratos/manager/dto/Volume.java

```diff
@@ -0,0 +1,43 @@
+package org.apache.stratos.manager.dto;
+
+/**
+ * Created by nishadi on 11/24/15.
+ */
```

Comment:
The  author name is removed and the license header is added in the commit a5416b7